AccessCheckoutSDK/scripts/mocks-build-phase.sh

```diff
@@ -0,0 +1,20 @@
+echo "Running from sh file"
+# Define output file. Change "${PROJECT_DIR}/${PROJECT_NAME}Tests" to your test's root source folder, if it's not the default name.
+OUTPUT_FILE="${PROJECT_DIR}/${PROJECT_NAME}Tests/GeneratedMocks.swift"
+OUTPUT_DIR="${PROJECT_DIR}/${PROJECT_NAME}Tests/mock/"
+echo "Generated Mocks File = ${OUTPUT_FILE}"
+
+# Define input directory. Change "${PROJECT_DIR}/${PROJECT_NAME}" to your project's root source folder, if it's not the default name.
+INPUT_DIR="${PROJECT_DIR}/${PROJECT_NAME}"
+echo "Mocks Input Directory = ${INPUT_DIR}"
+
+# Generate mock files, include as many input files as you'd like to create mocks for.
+"${PODS_ROOT}/Cuckoo/run" generate --testable "${PROJECT_NAME}" \
+--file-prefix "Mock" \
+--output "${OUTPUT_DIR}" \
+"${INPUT_DIR}/CardView.swift" \
+"${INPUT_DIR}/CVVValidator.swift" \
+"${INPUT_DIR}/cvvOnly/CVVOnlyDelegate.swift"
+# ... and so forth, the last line should never end with a backslash
+
+# After running once, locate `GeneratedMocks.swift` and drag it into your Xcode test target group.
```

Comment:
This is incorrect, because mocks are generated in individual files. I'll remove that line

Reply:
Done.